server/test-utils/src/main/java/io/deephaven/server/test/FlightMessageRoundTripTest.java

```diff
@@ -1301,19 +1301,24 @@ private void testZonedDateTimeColumnWithFactor(
         }
     }
 
+    private Schema createDoubleArraySchema() {
+        final Field payload = new Field("", new FieldType(false, Types.MinorType.FLOAT8.getType(), null), null);
+
+        final FieldType innerFieldType = new FieldType(true, Types.MinorType.LIST.getType(), null);
+        final Field inner = new Field("", innerFieldType, Collections.singletonList(payload));
```

Comment:
As part of https://github.com/deephaven/deephaven-core/issues/5497, I'd hoping we can consolidate this sort of building logic into a common place.

For example, should we have `innerFieldType` with `deephaven:type` "double[]"? Should `payload` have type `deephaven:type` "double"? It might be extraneous today, but may be helpful in the future?